charts/handbrake/0.0.1/test_values.yaml

```diff
@@ -0,0 +1,91 @@
+image:
+  repository: jlesage/handbrake
+  tag: v1.23.1
+  pullPolicy: IfNotPresent
+
+strategy:
+  type: Recreate
+
+service:
+  port:
+    port: 5800            #Required for web GUI
+appAdditionalServicesEnabled: true
+appAdditionalServices:
+  vnc:
+    enabled: true
+    type: ClusterIP
+    port:
+      port: 5900
```

Comment:
It's prefered to move this to the bottom for now...
In the future I will restructure this, but will do so for all charts in one go (and having one chart put it here makes it harder to restructure)